rinex/pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[build-system]
+requires = ["maturin>=0.14"]
+build-backend = "maturin"
+
+[project]
+name = "rinex"
+requires-python = ">=3.9"
```

Comment:
any particuliar reason for the `>=3.9` contraint ?

Reply:
No, not other than that was the version I had installed when I wrote it.

Reply:
If you don't mind I decremented to v3.8, the wheels are auto generated for 3.8, 3.9 and 3.10 conveniently

Reply:
That's perfectly fine. A lot of people still use 3.7 as far as I know.